### ISSUE TITLE
Composable multiple typeahead select: VO issue

### ISSUE THREAD

**Issue by kmcfaul** [patternfly/patternfly-react#8125]
The newer composable multiple typeahead select demo isn't quite working with VO and needs an update. It will require `aria-activedescendant` most likely or something similar to the original Select.

**Comment by jessiehuff** [comment]
So I'm not quite sure what has changed, but it seems like the options are getting announced for me now. Or perhaps I was using a different screen reader than VO or I saw something that I'm not seeing right now. I did notice that I wasn't able to escape/close the menu when going through the options though. @nicolethoen Would you be able to test out this example and see what you think of the interaction in general?

**Comment by nicolethoen** [comment]
@jessiehuff Options are getting announced when the VO focus is in the menu, they are not being announced as I type. It seems like a tough VO interaction.

I wish:

- When initially focusing on the input, the current list of selected items should be read - i dont know if that's possible.
- when I've already selected an option, it should be read as 'selected' when using VO to navigate through the list of options.

Do we already have an issue open for the single select typeahead demo? it appears that the menu is appended to the DOM and i cannot get screen reader focus in the menu.

**Comment by thatblindgeye** [comment]
Also look into Select Next implementation.
